plugins/modules/system/sysupgrade.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Andrew Klaus <andrewklaus@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
```

Comment:
Remove line 9 to 13 since these lines are no longer required for module code. Please refer - https://github.com/ansible-collections/overview/issues/45#issuecomment-627922948